### ISSUE TITLE
[BUG] Can't add OrgnalR Nuget package to Service Fabric service (stateless, ASP.NET Core web application)

### ISSUE THREAD

**Issue by danvanderboom** [LiamMorrow/OrgnalR#13]
**Describe the bug**
Nuget package installation fails and rolls back. There's a conflict with this package which is used by the generated code for a new project: Microsoft.ServiceFabric.AspNetCore.Kestrel.

**To Reproduce**
1. Create a new Service Fabric Application in Visual Studio.
2. From the cluster project (orange icon): right click, Add, "New Service Fabric Service...".
3. Select "Stateless ASP.NET Core", provide a name "Test.SignalR", and hit the Create button.
4. Attempt to add the SignalR.Orleans Nuget package to the new ASP.NET Core service project.

**Expected behavior**
The Nuget package is added and can be used.

**Comment by LiamMorrow** [comment]
Thanks for the bug report @danvanderboom.  Very interesting.  I'll have a dig tonight to see what the issue is.  Hopefully can push out a version that's compatible

**Comment by LiamMorrow** [comment]
I am able to reproduce... This isn't the first time I've run into issues with this System.Text.Encoding package (not in OrgnalR but professionally).  Will keep you in the loop

**Comment by LiamMorrow** [comment]
So.  I thought it might be that I should be targeting `netcoreapp3.1` and `netstandard2.0`, however this did not solve the issue.

I then went and installed `Microsoft.AspNetCore.SignalR.Core` - Worked
Installed `Microsoft.Orleans.Runtime.Abstractions`  - Failed

Tried both combinations.  So it seems like an issue when `Microsoft.ServiceFabric.AspNetCore.Kestrel` is installed with BOTH of those packages.

To be honest, I think the `Microsoft.AspNetCore.SignalR.Core` package is overdue for an update (if only to fix issues like this).  It hasn't been updated since 2018.  I'm gonna try and deploy a version which fixes the conflicts, but I have a feeling it won't be solvable in OrgnalR, and in your application you will need to go through and install each conflicting package explicitly
